docs/resources/dw_vw_impala.md.tpl

```diff
@@ -0,0 +1,26 @@
+---
+page_title: "{{.Name}} {{.Type}} - {{.ProviderName}}"
```

Comment:
templates/resources this file is not at the right place! Please, I linked the path where you have to add this in a previous comment.

Reply:
you can remove this file

Reply:
Thanks for the review. Done !

Reply:
I removed this file and committed but I found that when I reran make generate docs, this file shows up automatically